beryllium/mtcbench/cperf_ci.sh

```diff
@@ -6,25 +6,32 @@
 # terms of the Eclipse Public License v1.0 which accompanies this distribution,
 # and is available at http://www.eclipse.org/legal/epl-v10.html
 
-# Usage: ./cperf_ci
+# Usage: ./cperf_ci TEST_FILE.json
 # Prerequisites:
 # - docker
 # - docker-compose
-
+TEST_FILE=$1
+CONFIG_FILENAME=`echo "$TEST_FILE" | cut -d'.' -f1`
 NSTAT_WORKSPACE=/opt/nstat
 DOCKER_WORKSPACE=/opt/nstat
-CONFIG_FILENAME="beryllium_RPC_sb_active_scalability_mtcbench"
 RESULTS_DIR=$CONFIG_FILENAME"_results_"
 
+TMP=${CONFIG_FILENAME#*_*_}
+TEST_TYPE=${TMP%.*}
+echo '-------------------------------------------------------------------------'
+echo 'TEST TYPE      : '$TEST_TYPE
+echo 'CONFIG_FILENAME: '$CONFIG_FILENAME
+echo '-------------------------------------------------------------------------'
+
 mkdir $DOCKER_WORKSPACE
```

Comment:
we don't need lines 26-29, as we won't be downloading the docker-compose.yml from the web, we will directly find it in the current dir